app/views/shared/views/_prov_dialog.html.haml

```diff
@@ -180,7 +180,7 @@
         :prefix                     => "/miq_request/",
         :keys                       => keys})
   - when :customize
-    - if wf.kind_of?(ManageIQ::Providers::Vmware::InfraManager::ProvisionWorkflow) && !wf.supports_pxe? && !wf.supports_iso?
+    - if (wf.kind_of?(ManageIQ::Providers::Vmware::InfraManager::ProvisionWorkflow) || wf.kind_of?(ManageIQ::Providers::Redhat::InfraManager::ProvisionWorkflow)) && !wf.supports_pxe? && !wf.supports_iso?
```

Comment:
this looks like a pretty complex condition to be in a template, can you extract that to a (helper) method, please?

Reply:
btw does this condition represent some property of the provider? or the workflow? maybe it shoudl be pushed even deeper in the model?

Reply:
This condition checks only the type of the workflow. We want to have similar customize tab as there is for vmware.